genkit-tools/common/src/server/router.ts

```diff
@@ -239,6 +244,17 @@ export const TOOLS_SERVER_ROUTER = (manager: RuntimeManager) =>
         return response;
       }),
 
+    /** Start new evaluation run */
+    validateSchema: loggedProcedure
+      .input(apis.ValidateDataRequestSchema)
```

Comment:
the trpc method sounds generic, first I saw it I though "validate any arbitrary data against a schema"... but then it turned out it's eval dataset specific. Maybe call this `validateDatasetSchema`?